### ISSUE TITLE
need option on create container -ssh which enables ssh by default

### ISSUE THREAD

**Issue by despiegk** [threefoldtecharchive/zosctl#16]

**Comment by xmonader** [comment]
fixed in 8fd37055f96d4af06dd3f6080e91fb5be5eb7bef

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:
6cb6715360648ba3cce8ef22bb390025024d17b9

# Verification steps:

- zerotier-cli join 9bee8941b5717835
- ./zos container new  --name="test"  --root="https://hub.grid.tf/tf-bootable/ubuntu:lts.flist"

``` 
INFO new container: corex.create {"name":"test","hostname":"test","root":"https://hub.grid.tf/tf-bootable/ubuntu:lts.flist","privileged":false,"nics":[{"type":"default"},{"type":"zerotier","id":"9bee8941b5717835"}],"config":{"/root/.ssh/authorized_keys":"ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDHm7tHFg8+I6fPjyTOzMHF/MvZ7p8xyiDFIogFqcuIBoiyMSFaxPrhXZg4E3z73eGZ+o/axQwbdPtpszjzCjgtXvzLAaEdr4cKn9r78WUQP9egH/QS4XxA+qf8a8kQA4VojP7pb5G5P0HNfX5IgM3gRHJPq8pGBYgstI7T+q+bGsDvn29Y0AFMUbtKS6R24L9EZ+Ns5L/uWjWkIXh9Fab93rtGWoygscKIoWEfi8VHYjvl1JIuWHL6bTePTL5Q7co5ospGKEaiIysSM1vhol8uv1Mphp2oarQNdYhcva4IFsbPw89x7FIdOlzYkFZp/MKKEuHGK5w1tvUhkxtc9yxl root@dina-Inspiron-5423\n"}}

```

- ./zos container 2 sshenable

```
ssh root@10.244.250.236 -i /root/.ssh/id_rsa
```

- ssh root@10.244.250.236 -i /root/.ssh/id_rsa

```
The authenticity of host '10.244.250.236 (10.244.250.236)' can't be established.
ECDSA key fingerprint is SHA256:XpTFYT8lGEGIB39ExFKVLnN85JjXnC//HCGBYKmEYQc.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '10.244.250.236' (ECDSA) to the list of known hosts.
Welcome to Ubuntu 16.04 LTS (GNU/Linux 4.14.36-Zero-OS x86_64)
Documentation:  https://help.ubuntu.com/
The programs included with the Ubuntu system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.
Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
applicable law.
root@test:~# 
```
or 
- ./zos container shell
